logger/configurer/configurer.go

```diff
@@ -0,0 +1,137 @@
+package configurer
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/coreos/go-etcd/etcd"
```

Comment:
For better or apparently worse, we already have `go-etcd` vendored into the project, but not the newer client.

> Maybe we should stay with this one for now and switch for version 2.

I think that's preferable, then switch over completely to the shinier package either in a 1.12 or 2.0 release.